src/components/ebay-table/component.ts

```diff
@@ -1,10 +1,14 @@
-import { AttrTriState } from "marko/tags-html";
+import { AttrString, AttrTriState } from "marko/tags-html";
 import { WithNormalizedProps } from "../../global";
 import { CheckboxEvent } from "../ebay-checkbox/component-browser";
 
 type TableColRowName = string | number;
```

Comment:
I know this isn't in this PR, but can we pick a better name?
Is this the name of the row or columns?

Maybe just TableRowName or TableRowIdentifier?

Reply:
Maybe force it to be `string` and just not to add this type at all? `string | number` just makes it easy for index-as-name, but we can cast it to string as well.